src/headless/utils/url.js

```diff
@@ -21,7 +21,7 @@ export function getURI (url) {
 
 function checkFileTypes (types, url) {
     const uri = getURI(url);
-    if (uri === null || !checkTLS(uri)) {
+    if (window.location.protocol !== 'chrome-extension:' && (uri === null || !checkTLS(uri))) {
```

Comment:
Looking at this again, I think this needs to be changed to:

`(uri === null || (window.location.protocol !== 'chrome-extension:' && !checkTLS(uri)))`

Otherwise, in a chrome extension, if `uri` is `null` a `TypeError` will be thrown on line 27.